src/payment/create-payment-strategy-registry.ts

```diff
@@ -321,6 +321,16 @@ export default function createPaymentStrategyRegistry(
         )
     );
 
+    registry.register(PaymentStrategyType.PAYPAL_COMMERCE_ALTERNATIVE_METHODS, () =>
+        new PaypalCommercePaymentStrategy(
+            store,
+            orderActionCreator,
+            paymentActionCreator,
+            createPaypalCommercePaymentProcessor(scriptLoader, requestSender, true),
```

Comment:
instead of having a third param to simply be `true`, I think it would be easier to read and evolve if you were to pass a configuration object, like
```ts
createPaypalCommercePaymentProcessor(scriptLoader, requestSender, {
  provider: PaypalCommerceType.ALTERNATIVE_METHODS,
}),
```

Reply:
fixed